src/components/app/data/utils.test.js

```diff
@@ -990,3 +996,45 @@ describe('transformLearnerContentAssignment', () => {
     expect(transformedAllocatedAssignment.courseRunId).toEqual(contentKey);
   });
 });
+describe('resolveBFFQuery', () => {
+  const dashboardRoute = '/:enterpriseSlug';
+  const routes = [dashboardRoute];
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+  it('returns the dashboard query key', () => {
+    const pathname = '/testEnterpriseSlug';
+    const mockParams = { enterpriseSlug: 'testEnterpriseSlug' };
+    const expectedQueryKey = [
+      'enterprise',
+      'enterpriseCustomer',
+      null,
```

Comment:
nit: does this represent the empty customer `uuid`? May make sense to leave a helper comment describing the intended `null`. Alternatively, would it be worth considering using the query key helper function here?